docs/modules/debug/require-inspector.md

```diff
@@ -0,0 +1,28 @@
+<%( TOC )%>
+
+# Require inspector extension
+
+The Require inspector extension is a debugging extension for the Unity inspector, which helps with visualizing the use of `[Require]` in MonoBehaviour components.
+
+## Usage
+
+To use the extension, ensure the package is added to your project and enter Play mode.
+
+1. Select any SpatialOS Gameobject in your Hierarchy.
```

Comment:
```suggestion
1. Select any Unity GameObject that is linked to a SpatialOS entity in your Hierarchy.
```

Reply:
should have a mention somewhere that this only works together with the GameObject module and some docs to there, if they want to find to out more

Reply:
It works with the subscription system, which is core?

Reply:
I meant that you can only use it when you have a gameobject linked to a SpatialOS entity -> need to use the gameobject module, wihch is at the moment core.
but maybe adding a link to this point is enough

Reply:
The linking is actually part of the Core right now - https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/io.improbable.gdk.core/Subscriptions/EntityGameObjectLinker.cs 😉

Reply:
This actually raises an interesting question for the future though. When gameobject linking moves out of the Core what does the Debug module depend on? 

Do we have the tooling inside each feature module or bunch up tooling into these Debug feature modules?